db/seeds.d/90_add_permissions_from_default_roles.rb

```diff
@@ -1,12 +1,14 @@
 default_permissions = Foreman::Plugin.find("foreman_discovery").default_roles
```

Comment:
nitpick, all strings in here can safely be frozen, you can add

```
# frozen_string_literal: true
```

at the beginning of the file for better performance & thread safety (I know it's a little ridiculous for a loop with 2 objects but it's easy enough)